package.json

```diff
@@ -75,11 +77,11 @@
   ],
   "scripts": {
     "check-version": "node test/check-version.js",
-    "tslint": "tslint src/*.ts",
+    "lint-code": "eslint src/*.ts",
```

Comment:
I'd call it `lint`, but that's personal preference.

Reply:
I choose `lint-code` because there is a script called `format-code`.
@ariya what do you think?